# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/linux/linux26/82-raid5-niceer-output.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./crypto/xor.c.orig	2004-08-14 12:55:32.000000000 +0200
+++ ./crypto/xor.c	2004-08-17 18:57:24.679307424 +0200
@@ -117,6 +117,8 @@
 
 #ifdef XOR_SELECT_TEMPLATE
 		fastest = XOR_SELECT_TEMPLATE(fastest);
+	 	printk("raid5: using %s to write around L2 cache\n",
+	 		fastest->name);
 #endif
 
 #define xor_speed(templ)	do_xor_speed((templ), b1, b2)
